lib/rubocop/cop/style/ascii_comments.rb

```diff
@@ -5,6 +5,12 @@ module Cop
     module Style
       # This cop checks for non-ascii (non-English) characters
       # in comments.
+      # @example
+      #   # bad
+      #   # 🤔
```

Comment:
A better example would be some text that's partially English and partially something else I guess.